systemd/docker-novolume-plugin.socket

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Docker novolume plugin Socket for the API
+#PartOf=docker-novolume-plugin.service
```

Comment:
@rhatdan @lsm5 I've removed this line cause we don't want to stop the systemd socket if the plugin .service dies (right?)